script.js

```diff
@@ -159,3 +159,43 @@ contactForm.addEventListener('submit', (event) => {
     event.preventDefault();
   }
 });
+
+// Session storage
+
+const sessionObj = {
+  fullName: '',
+  email: '',
+  textArea: '',
+};
+
+const fullName = document.querySelector('#footer-name');
+const textArea = document.querySelector('#message');
+let str;
+
+if (localStorage.test === undefined) {
+  localStorage.setItem('test', JSON.stringify(sessionObj));
+}
+const testValue = localStorage.getItem('test');
+const testValueObj = JSON.parse(testValue);
+fullName.value = testValueObj.fullName;
+email.value = testValueObj.email;
+textArea.value = testValueObj.textArea;
+
```

Comment:
- [ ] [OPTIONAL] Please fix the following behavior:
   - When the user writes data in the inputs field, the data is preserved correctly in the local storage. However, when the user tries to add to the data imported from the local storage, the local storage objects get overwritten with the empty object and you lose the data preserved. 
 ### Hint: You can update the `sessionObj` object with data in the local storage like:
 ```
sessionObj.fullName = testValueObj.fullName;
sessionObj.email = testValueObj.email;
sessionObj.textArea = testValueObj.textArea;
```

Reply:
Okay, thanks for the feedback and suggestions!

Reply:
Thanks for the feedback!